src/components/employeesCard/EmployeeCard.module.scss

```diff
@@ -31,11 +32,14 @@
     .headerBackground {
       position: absolute !important;
       z-index: -1;
-      height: 100px;
-      width: calc(
-        565px - 140px
-      ); // Width is card width minus image-column-width
+      height: 100%;
+      width: 100%;
       background-color: $darkBlue; // Fallback if background pattern doesn't load
+
+      img {
+        height: 500% !important;
```

Comment:
Hvorfor trenger vi 500% og important her ?

Reply:
important fordi det overstyrer Gatsby-Image. 500% brukte jeg for å skalere mønsteret, men burde kanskje endret selve bildet i stedet.